test/torchaudio_unittest/transducer_test.py

```diff
@@ -241,6 +242,14 @@ def compute_with_pytorch_transducer(data):
     return costs, gradients
 
 
+def skipIfNoRNNT(test_item):
+    try:
+        _ = torch.ops.torchaudio.rnnt_loss
```

Comment:
nit: i see a [discussion above](https://github.com/pytorch/audio/pull/1253#discussion_r572971066) adding `_ = ...`, but `_ = ` isn't needed, right?
```suggestion
        torch.ops.torchaudio.rnnt_loss
```